Subject: [PATCH] Demo prep
---
Index: src/main/java/org/acme/Filter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/acme/Filter.java b/src/main/java/org/acme/Filter.java
--- a/src/main/java/org/acme/Filter.java	(revision cfe55b9c809dd56d14da9c079acc1dda82d510b8)
+++ b/src/main/java/org/acme/Filter.java	(date 1762260188468)
@@ -1,12 +1,10 @@
 package org.acme;
 
 import jakarta.ws.rs.container.ContainerResponseContext;
-import org.jboss.resteasy.reactive.server.ServerResponseFilter;
 
 import static org.acme.support.Translator.translate;
 
 public class Filter {
-    @ServerResponseFilter
     public void getFilter(ContainerResponseContext responseContext) {
         Object entity = responseContext.getEntity();
         if (entity instanceof String s) {
